packages/gatsby-plugin-sass/README.md

```diff
@@ -26,15 +27,16 @@ html {
 ```
 
 ```javascript
-import("./src/index.sass")
+// in gatsby-browser.js
+import("./src/index.scss")
 ```
 
 ## Other options
 
 If you need to pass options to Sass use the plugins options, see [node-sass](https://github.com/sass/node-sass)/[dart-sass](https://github.com/sass/dart-sass) docs
 for all available options.
 
-```javascript
+```js
```

Comment:
Both `js` and `javascript` are used throughout the docs, and your later entry uses `javascript`...perhaps we should make them consistent, at least in the same file?